code/modules/shuttle/computer.dm

```diff
@@ -20,17 +20,132 @@
 	/// Authorization request cooldown to prevent request spam to admin staff
 	COOLDOWN_DECLARE(request_cooldown)
 
+	var/uses_overmap = TRUE //SKYRAT EDIT ADDITION
+
 /obj/machinery/computer/shuttle/Initialize(mapload)
 	. = ..()
 	if(!mapload)
 		connect_to_shuttle(SSshuttle.get_containing_shuttle(src))
 
+//SKYRAT EDIT ADDITION
+/obj/machinery/computer/shuttle/attacked_by(obj/item/smacking_object, mob/living/user)
+	if(istype(smacking_object, /obj/item/navigation_datacard))
+		var/obj/docking_port/mobile/M = SSshuttle.getShuttle(shuttleId)
+		if(M.gateway_stranded)
+			to_chat(user, span_notice("You insert [smacking_object] into [src] loading the navigational data!"))
```

Comment:
```suggestion
			to_chat(user, span_notice("You insert [smacking_object] into [src], loading the navigational data!"))
```
punctuation is cool

Reply:
ok upon further review, i found no typos or issues, but have learned that code is scary